packages/fast-components-react-msft/src/action-toggle/README.md

```diff
@@ -0,0 +1,13 @@
+# Action Toggle
+*Action toggle* should be used as control on or around an interactive region of the page that affects the region or items within the region. It usually represents a switch between one of two states.
+
+## Usage
+Use *action toggle* for specific control around a functional state. Labeling can be done using a glyph and text, but could also contain just a glyph or just text. Make sure the text and glyph clearly represent the expected function. When only a glyph is used to label the button, include a tooltip in a span element following the button element.
+
+The "selectedGlyph" and "unselectedGlyph" props take a function that returns a ReactNode to render the glyph (ie. a [render prop](https://reactjs.org/docs/render-props.html)).  Note that the function should accept a string parameter that is applied as a classname to the top level element of the resulting node.  This classname  will be populated by the *action toggle* component to use an internally generated class ("actionToggle_selectedGlyph" & "actionToggle_unselectedGlyph") to apply to the glyphs for styling. 
```

Comment:
class name should be two words

Reply:
I'm also not sure we should be including this level of detail about something that is basically baked-in.

Reply:
"string parameter applied as a class name"

Reply:
I am seeing a lot of "resolved" comments without changes. For instance, above class name is still one word, yet this  was marked as resolved. If it is resolved, I would expect the change OR if there is a disagreement, there should be a follow-up comment regarding it.

Reply:
Because I have not yet pushed up an updated changelist yet?

Reply:
Updated with changes.